readers-frontend/src/components/Article.astro

```diff
@@ -25,7 +26,10 @@ const {
   articleUrl,
   isVertical,
   isPortrait,
+  isInMuseBanner,
 } = Astro.props;
+
+const imagePlaceHolder = '"https://fakeimg.pl/640x360"';
```

Comment:
is this because some articles don't have a placeholder yet? If so can you move it to the s3files where we're grabbing the article image please?

Reply:
No because this is for when loading the image fails. Not when there is no image on the db. This runs when we get an image link from the db but the actual blob is missing on the bucket

Reply:
I see that makes sense. Can you add a Todo around it to replace with an asset that we upload with the code please?